conventions/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -338,13 +358,17 @@ afterEvaluate {
       if (!isJavaVersionAllowed(DEFAULT_JAVA_VERSION) && otelJava.maxJavaVersionForTests.isPresent) {
         javaLauncher.set(
           javaToolchains.launcherFor {
-            languageVersion.set(JavaLanguageVersion.of(otelJava.maxJavaVersionForTests.get().majorVersion))
+            languageVersion.set(
+              JavaLanguageVersion.of(otelJava.maxJavaVersionForTests.get().majorVersion)
+            )
           }
         )
       }
     }
 
-    if (plugins.hasPlugin("org.unbroken-dome.test-sets") && configurations.findByName("latestDepTestRuntime") != null) {
+    if (plugins.hasPlugin("org.unbroken-dome.test-sets") && configurations.findByName(
+        "latestDepTestRuntime"
+      ) != null) {
```

Comment:
I can't say that I'm a fan of this kind of formatting -- should we increase the max line length for kotlin files? Maybe just for the gradle scripts

Reply:
👍

Reply:
Changed max line length for Kotlin files to 160 -- I think it looks much better now.